proto/common/server_enum.proto

```diff
@@ -24,47 +24,45 @@ package common;
 
 option go_package = "github.com/temporalio/temporal/.gen/proto/common";
 
-enum DLQType {
-    Replication = 0;
-    Namespace = 1;
+enum DeadLetterQueueType {
```

Comment:
Had to rename this type to make it work with CamelCase <-> SCREAMING_CASE conversion.